fieldsolver/gridGlue.cpp

```diff
@@ -276,13 +276,13 @@ void getFieldsFromFsGrid(
         sendBuffer[ii].sums[3 ] += volcell->at(fsgrids::volfields::EXVOL);
         sendBuffer[ii].sums[4 ] += volcell->at(fsgrids::volfields::EYVOL);
         sendBuffer[ii].sums[5 ] += volcell->at(fsgrids::volfields::EZVOL);
-        sendBuffer[ii].sums[6 ] += volcell->at(fsgrids::volfields::dPERBXVOLdy);
-        sendBuffer[ii].sums[7 ] += volcell->at(fsgrids::volfields::dPERBXVOLdz);
-        sendBuffer[ii].sums[8 ] += volcell->at(fsgrids::volfields::dPERBYVOLdx);
-        sendBuffer[ii].sums[9 ] += volcell->at(fsgrids::volfields::dPERBYVOLdz);
-        sendBuffer[ii].sums[10] += volcell->at(fsgrids::volfields::dPERBZVOLdx);
-        sendBuffer[ii].sums[11] += volcell->at(fsgrids::volfields::dPERBZVOLdy);
-        sendBuffer[ii].sums[12] += bgcell->at(fsgrids::bgbfield::BGBXVOL); 
+        sendBuffer[ii].sums[6 ] += volcell->at(fsgrids::volfields::dPERBXVOLdy) / technicalGrid.DY;
```

Comment:
Hmm, so the fsgrid now contains finite difference deltas, and the DCCRG grid actually contains derivatives here, right?

Not fundamentally bad, but potentially confusing, that these two variables with the same name have different semantics.

Reply:
Should this division be already be performed in the fieldsolver, when the values are written to fsgrid in the first place?

Reply:
Yes but that will introduce a zillion small changes to the field solver.

Reply:
That is a large change. If one wants to take care of it then instead rename derivative => difference or something. It is more of an naming problem. No sense to do it now for this fix

Reply:
The variable name in FSgrid is indeed confusing. Would it be too much to rename them all over:
fsgrids::volfields::dPERBYVOLdz -> fsgrids::volfields::dzPERBYVOL etc?

Reply:
It might also clog the view when we debug this at some point, but it's less risky (only a sed away) than manually changing the divisions by D[XYZ] indeed.

Reply:
Alright, this'll work just fine then.

We can consider *also* doing this division if we output this data though. But that's a separate construction site. :)

Reply:
This branch or with the sed suggested by @markusbattarbee?